src/selectors/entities/posts.js

```diff
@@ -278,11 +290,11 @@ export function makeGetPostsAroundPost() {
 // Returns a function that creates a creates a selector that will get the posts for a given thread.
 // That selector will take a props object (containing a rootId field) as its
 // only argument and will be memoized based on that argument.
-export function makeGetPostsForThread() {
+export function makeGetPostsForThread(): (GlobalState, {rootId: $ID<Post>}) => Array<Post> {
     return createSelector(
         getAllPosts,
-        (state, {rootId}) => state.entities.posts.postsInThread[rootId] || [],
-        (state, {rootId}) => state.entities.posts.posts[rootId],
+        (state: GlobalState, {rootId}) => state.entities.posts.postsInThread[rootId] || [],
```

Comment:
Not something for this PR, but these selectors aren't memoized properly because of the array literal here. I'll file a ticket for that